### ISSUE TITLE
Plugin submission error group 1: generic function names

### ISSUE THREAD

**Issue by jonathanstegall** [MinnPost/object-sync-for-salesforce#86]
As in #87, this is the email from the initial WordPress plugin review.

> There are issues with your plugin code.
> 
> Please read this ENTIRE email, address all listed issues, and reply to this email with your corrected code attached. It is required for you to read and reply to these emails, and failure to do so will result in significant delays with your plugin being accepted.
> 
> Also please remember in addition to code quality, security and functionality, we require all plugins adhere to our guidelines. If you have not yet, please read them:
> 
> * https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/

## Generic function (and/or define) names

All plugins must have unique function names, defines, and classnames. This prevents your plugin from conflicting with other plugins or themes.

For example, if your plugin is called "Easy Custom Post Types", then you might prefix your functions with ecpt_{your function name here}. Similarly a define of LICENSE would be better done as ECPT_LICENSE. You can use namespaces instead, however make sure that those also are unique. A namespace or class of 'MyPlugin' is NOT actually all that unique.

This extends to anything in a define. For example, if you were to use this, it would be a bad idea:

```
define( 'PLUGIN_PATH', plugins_url( __FILE__ ) );
```

That define is a global, so PLUGIN_PATH could conflict with a number of other things.

Don't try to use two letter slugs anymore. As of 2016, all the good ones are taken. Instead consider easy_cpts_ (from the first example).

Similarly, don't use __ (double underscores), wp_ , or _ (single underscore) as a prefix. Those are reserved for WordPress itself. You can use them inside your classes, but not as stand-alone function.

Please update your plugin to use more unique function and class names.

Some examples from your plugin:

```
class Admin_Notice
class Wordpress
```

----

> Please make sure you've addressed ALL issues brought up in this email. When you've corrected your code, reply to this email with the updated code attached as a zip, or provide a link to the new code for us to review. If you use gmail, you won’t be able to send a ZIP file if it contains any JS files (yes, we know it’s stupid, blame Google).
> 
> If you have questions, concerns, or need clarification, please reply to this email and just ask us.
> 
> (While we have tried to make this review as exhaustive as possible we, like you, are humans and may have missed things. As such, we will re-review the ENTIRE plugin when you send it back to us. > We appreciate your patience and understanding in this.)

**Comment by jonathanstegall** [comment]
Fixed in #88.

**Comment by jonathanstegall** [comment]
These changes were approved by the WordPress review people as well.
